src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -1089,3 +1090,9 @@ export const selectEquityTotalBalance = createDeepEqualOutputSelector(
     )
   },
 )
+
+export const selectWalletIdInStore = createSelector(
```

Comment:
This one is a bit odd naming-wise, I was going to suggest `selectWalletId` but we already have this guy:
https://github.com/shapeshift/web/blob/9d1978ca8e0b9ae3881e79ed5a6dd3a139a8b18c/src/state/slices/common-selectors.ts#L19

Since this effectively only checks for presence given a `walletId` predicate, how about `selectPortfolioHasWalletId`?